File: Project/admin-app/src/app/models/agent-dto.ts
export class AgentDTO {
    userToPromote: string;
    address: string;
    businessRegNumber: string;

    constructor(userToPromote: string, address: string, businessRegNumber: string) {
        this.userToPromote = userToPromote;
        this.address = address;
        this.businessRegNumber = businessRegNumber;
     }
}

File: Project/central-api/src/main/java/com/centralapi/domain/xml/xml_ftn/users/AgentDTO.java
package com.centralapi.domain.xml.xml_ftn.users;

public class AgentDTO {
	
	private String userToPromote;
	private String address;
	private String businessRegNumber;
	
	public AgentDTO() {}

	public String getUserToPromote() {
		return userToPromote;
	}

	public void setUserToPromote(String userToPromote) {
		this.userToPromote = userToPromote;
	}

	public String getAddress() {
		return address;
	}

	public void setAddress(String address) {
		this.address = address;
	}

	public String getBusinessRegNumber() {
		return businessRegNumber;
	}

	public void setBusinessRegNumber(String businessRegNumber) {
		this.businessRegNumber = businessRegNumber;
	}

}
